File: API/API/Modules/PaymentsModule/Model/DTO/PaymentPayerOwnDTO.cs
using API.Modules.InvoiceModule.Model.DTO;

namespace API.Modules.PaymentsModule.Model.DTO;

public class PaymentPayerOwnDTO
{
    public Guid Id { get; set; }
    public required float Money { get; set; }
    public required PaymentType Type { get; set; }
    public required DateTime DateTime { get; set; }
    public InvoiceDTO? Invoice { get; set; }
}

File: Client/angular-monorepo/apps/manager-app/src/app/modules/main-router/modules/payments/DTO/PaymentPayerOwnDTO.ts
import { PaymentType } from '../models/PaymentType.enum';
import { IInvoiceDTO } from '../../invoices/DTO/InvoiceDTO';


export interface IPaymentPayerOwnDTO {
  id: string;
  money: number;
  type: PaymentType;
  dateTime: string;
  invoice?: IInvoiceDTO;
}
